release/community-stable/clearlinux/docker/Dockerfile

```diff
@@ -95,13 +95,10 @@ RUN \
             Start-Sleep -Seconds 6 ; \
           }" \
     # upgrade distro
-    && swupd update \
-    # clean downloaded packages
-    && swupd clean --all
+    && swupd update
 
 # Copy only the files we need from the previous stage after distro upgrade
 # These files are required to resolve SSL issue with Invoke-WebRequest
-COPY --from=installer-env ["/usr/lib64/libcrypto.so.1.0.0", "/usr/lib64/libcrypto.so.1.0.0"]
```

Comment:
@RDIL The libcrypto.so.1.0.0 is needed

Reply:
The CI can't find it in the location specified